algorithm_rgb.py

```diff
@@ -1,62 +1,184 @@
-"""My nifty plot-level RGB algorithm
+"""Greenness Transformer
 """
 
-# Importing modules. Please add any additional import statements below
+# Importing modules.
 import numpy as np
 
 # Definitions
-# Please replace these definitions' values with the correct ones
 VERSION = '1.0'
 
 # Information on the creator of this algorithm
-ALGORITHM_AUTHOR = 'Unknown'
-ALGORITHM_AUTHOR_EMAIL = ''
-ALGORITHM_CONTRIBUTORS = [""]
-
-ALGORITHM_NAME = 'my nifty one'
-ALGORITHM_DESCRIPTION = 'This algorithm calculates the niftyness of RGB plot-level images'
-
-# Citation information for publication (more information in HOW_TO.md)
-CITATION_AUTHOR = 'unknown'
-CITATION_TITLE = ''
-CITATION_YEAR = ''
-
-# The name of one or more variables returned by the algorithm, separated by commas (more information in HOW_TO.md)
-# If only one name is specified, no comma's are used.
-# Note that variable names cannot have comma's in them: use a different separator instead. Also,
-# all white space is kept intact; don't add any extra whitespace since it may cause name comparisons
-# to fail.
-# !! Replace the content of this string with your variable names
-VARIABLE_NAMES = 'size of image channels'
+ALGORITHM_AUTHOR = 'Chris Schnaufer, Clairessa Brown, David Lebauer'
+ALGORITHM_AUTHOR_EMAIL = 'schnaufer@arizona.edu, clairessabrown@email.arizona.edu, dlebauer@email.arizona.edu'
+ALGORITHM_CONTRIBUTORS = ["Jacob van der Leeuw"]
+
+ALGORITHM_NAME = 'Greenness Transformer'
+ALGORITHM_DESCRIPTION = 'This algorithm performs a variety of calculations using RGB pixels from images in order' \
+                        'to assess plant and crop health and growth'
+
+# Citation information for publication
+CITATION_AUTHOR = 'Clairessa Brown'
+CITATION_TITLE = 'Woebbecke, D.M. et al'
+CITATION_YEAR = '2020'
+
+# The name of one or more variables returned by the algorithm, separated by commas
+
+VARIABLE_NAMES = 'excess greenness index, green leaf index, cive, normalized difference index, excess red, ' \
+                 'exgr, combined indices 1, combined indices 2, vegetative index, normalized green-red difference,' \
+                 ' percent green'
 
 # Variable units matching the order of VARIABLE_NAMES, also comma-separated.
-# For each variable name in VARIABLE_NAMES add the unit of measurement the value represents.
-# !! Replace the content of this string with your variables' unit
-VARIABLE_UNITS = 'pixels'
+VARIABLE_UNITS = '[-510:510], [-1:1], [-255:255], [-127:129], [-255:255], [-255:332], ' \
+                 '[-1000:1000], [-1000:1000], [-255:255], [-255:255], [0:100]'
 
 # Variable labels matching the order of VARIABLE_NAMES, also comma-separated.
-# This is an optional definition and can be left empty.
-VARIABLE_LABELS = ''
+VARIABLE_LABELS = 'excess_greenness_index, green_leaf_index, cive, normalized_difference_index(pxarray), ' \
```

Comment:
why is the variable named 'normalized_difference_index(pxarray)' when it's a float? Shouldn't `(pxarray)` be removed?

Reply:
just removed